src/components/action-sheet/action-sheet.html

```diff
@@ -1,4 +1,4 @@
 <div class="backdrop" [ngClass]="{'backdrop-fade-in': slideIn}" (click)="dismiss()"></div>
-<div class="action-sheet" [ngClass]="{'opened': slideIn, 'closed-top': !slideIn && fromTop, 'closed-bottom': !slideIn && !fromTop, 'action-sheet-top': fromTop, 'action-sheet-bottom': !fromTop }">
+<div class="action-sheet" [ngClass]="{'opened': slideIn, 'closed-top': !slideIn && fromTop, 'closed-bottom': !slideIn && !fromTop, 'action-sheet-top': fromTop, 'action-sheet-bottom': !fromTop, 'safe-top-area': fromTop }">
```

Comment:
it's too complex to read. Why not group class with same condition?

Reply:
you're right!